internal/store/postgres/migrations/20230410103624_create_flows_table.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS flows(
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    method varchar,
```

Comment:
instead of `varchar` can we use `text`

Reply:
isn't `VARCHAR` and `TEXT` same if we don't specify the number to limit the characters?

https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-char-varchar-text/#:~:text=Different%20from%20other%20database%20systems,to%20check%20for%20the%20length.

Reply:
Yes they are same, I thought in rest of the migrations we are using `text` so I thought we may want to keep it consistent, I just checked looks like we actually use `varchar`. Should be fine then.